lib/index.js

```diff
@@ -343,3 +372,13 @@ function _updateSessionAuthenticationData(
   }
   return data;
 }
+
+function _toBase64NoPad(x) {
+  return x.toString('base64', 0, 16).split('=')[0];
+}
+
+function _sleep({delay}) {
+  return new Promise(resolve => {
+    setTimeout(resolve, delay);
+  });
```

Comment:
Both of these methods `_toBase64NoPad` and `_sleep` are reduced (or can be reduced) to single lines (and neither does anything crazy complicated). The functions themselves are both only used once so I don't see the point of keeping them as separate functions.